### PR TITLE
Adding section to identify projects without an owner

### PR DESCRIPTION
**Pull request by gsnyder2007**
Also fixing an issue where 'messages' were over-writing one another at the project, version, and scan levels.

### CODE REVIEW

File: sage.py [hunk 1]
```diff
@@ -14,15 +14,14 @@
 # TODO: Find scans (code locations) whose scan frequency is higher than we recommend
 # TODO: Find signature scans taking a long time to complete (e.g. > 30m ) and suggest they be optimized, e.g. by splitting things up
 # TODO: Find signature scans that are particular large and should possibly be split up
-# TODO: Find projects that don't have any Released versions
```
Comment (skiyooka) [inline sage.py:17]: Did you mean to remove this line?  And/Or the one two line below?

File: sage.py [hunk 2]
```diff
@@ -14,15 +14,14 @@
...
 # TODO: Find Released versions that don't have any reports
 # TODO: Find projects that don't have an owner
```
Comment (skiyooka) [inline sage.py:18]: The change adds functionality corresponding to this line.

File: sage.py (hunk 1 above)
Comment (gsnyder2007) [reply sage.py:17]: i already added something to identify project-versions that are marked as Released so I figured that's enough?

File: sage.py (hunk 2 above)
Comment (gsnyder2007) [reply sage.py:18]: i already added something to identify project-versions that are marked as Released so I figured that's enough?

**Comment by skiyooka** [comment]
Closing this request as I replaced this with #16

**Pull request closed without merging**